spec/factories/references.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :reference do
     application nil
+    sequence(:reference) { |n| "AB001-#{Time.zone.now.strftime('%y')}-#{n}" }
   end
 end
```

Comment:
Do we still need this factory?  
The reference model was deleted and the application factory has the sequence code now.